examples/bouss/setenv.sh

```diff
@@ -3,15 +3,39 @@
 # to run the Bouss version of GeoClaw with MPI and OpenMP.
 # Adjust as needed for your system...
 
-# You also need to set CLAW, FC, and perhaps PYTHONPATH
-
 # For more information, see
 #   https://www.clawpack.org/bouss2d.html
 #   https://www.clawpack.org/setenv.html
 
-export PETSC_DIR=/full/path/to/petsc
-export PETSC_ARCH=arch-darwin-c-opt
-export PETSC_OPTIONS="-options_file $CLAW/geoclaw/examples/bouss/petscMPIoptions"
+# You also need to set CLAW and perhaps PYTHONPATH, see:
+#   https://www.clawpack.org/setenv.html
+
+echo CLAW is set to $CLAW
+
+# path to PETSc installation:
+export PETSC_DIR=/full/path/to/petsc  # NEED TO FIX!
+
+# PETSC_ARCH is only needed if PETSc is installed inside the PETSc directory.
+# For PETSc installs by conda or package managers, it should not be set.
+#export PETSC_ARCH=
+export PETSC_ARCH=arch-darwin-c-opt  # NEED TO FIX!
+
+# You may want to use a different version of petscMPIoptions
+# This setting uses the version in this directory:
+export PETSC_OPTIONS="-options_file $PWD/petscMPIoptions"
```

Comment:
I was thinking the user would execute this file from within this directory, but I agree that it would be clearer to put in the full path so they see how to adjust it, and in case they want to execute it from their own example directory.  Thanks!